modules/v2/metadata.yaml

```diff
@@ -141,6 +141,12 @@ spec:
             }))
         required: true
         connections:
+          - source:
+              source: github.com/terraform-google-modules/terraform-google-memorystore/tree/main/modules/redis-cluster
+              version: ">= 12.0"
+            spec:
+              outputExpr: "{\"REDIS_CLUSTER_ENDPOINT\":discovery_endpoints}"
```

Comment:
What is the type of `discovery_endpoints`? The env_vars key and value can only be of string type.